types/dispatcher.d.ts

```diff
@@ -143,4 +144,17 @@ declare namespace Dispatcher {
     onBodySent?(chunkSize: number, totalBytesSent: number): void;
   }
   export type PipelineHandler = (data: PipelineHandlerData) => Readable;
+
+  /** 
+   * @link https://fetch.spec.whatwg.org/#body-mixin
+   */
+  interface BodyMixin {
+    readonly body?: never // throws on node v16.6.0
```

Comment:
What does it throw?

Reply:
```
      this[kBody] = Readable.toWeb(this)
                             ^

TypeError: Readable.toWeb is not a function
```
and @types/node are not updated to v16 yet so they don't include a web ReadableStream type sadly.